iotilecore/iotile/core/dev/annotated_registry.py

```diff
@@ -80,6 +81,15 @@ def list_config(self):
 
         return self.reg.list_config()
 
+    @param("device_uuid", "int", desc="Device UUID")
+    @param("password", "str", desc="Password to the device")
+    def set_temporary_password(self, device_uuid: int, password: str):
+        InMemoryAuthProvider.add_password(device_uuid, password)
```

Comment:
There should be docstrings for this since that sets the `help` text when running `help set_temporary_password` in the iotile shell.

Also, all new annotated functions should use `@docannotate` instead of `@param`.  That removes the need to duplicate `desc=` strings and improves how the help text looks.

Reply:
Got it. Fixed it. Can I fix the rest of this file? Or are there reasons we would want to have those remain as is?

Reply:
No reason they need to remain as is, if it's easy to update.

Reply:
Cool, updated it.